src/modules/credits/sagas.spec.ts

```diff
@@ -0,0 +1,124 @@
+import { expectSaga } from 'redux-saga-test-plan'
+import { call, select } from 'redux-saga-test-plan/matchers'
+import { throwError } from 'redux-saga-test-plan/providers'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { CONNECT_WALLET_SUCCESS } from 'decentraland-dapps/dist/modules/wallet/actions'
+import { Wallet } from 'decentraland-dapps/dist/modules/wallet/types'
+import { fetchCreditsRequest, fetchCreditsSuccess, fetchCreditsFailure, pollCreditsBalanceRequest } from './actions'
+import { creditsSaga, creditsService } from './sagas'
+import { getCredits } from './selectors'
+import { CreditsResponse } from './types'
+
+jest.mock('../vendor/decentraland/credits/api', () => ({
```

Comment:
This seems to have been left from the previous implementation, would you mind removing it?